runtime/proto/corfu_options.proto

```diff
@@ -19,6 +19,8 @@ message SchemaOptions {
     repeated string stream_tag = 5;
     // Nested Secondary Key (repeated field)
     repeated NestedSecondaryIndex nested_secondary_key = 6;
+    // Should the entry of table be validated ownership
```

Comment:
this comment is a bit unclear... what does the option mean?

Reply:
his flag will indicate whether the shim-layer will make a call to validate the ownership of the DB entry.

Reply:
ok, can we say something like 'should the entry.. validated for ownership'?

Reply:
sure, will do

Reply:
Done